### PR TITLE
Mixed precision tests coverage - sensitivity evaluation methods coverage

### PR DESCRIPTION
**Pull request by ofirgo**
- Add test to output replacement nodes for mixed precision gradients weights computation (tf and pytorch)
- Removed _tensor_to_list from MP models output

### CODE REVIEW

File: tests/keras_tests/feature_networks_tests/feature_networks/mixed_precision_tests.py [hunk 1]
```diff
@@ -514,3 +514,35 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info:
...
+        inputs = layers.Input(shape=self.get_input_shapes()[0][1:])
+        x = layers.Conv2D(32, 4)(inputs)
+        x = layers.BatchNormalization()(x)
+        x = layers.Conv2D(32, 4)(x)
+        x = layers.ReLU()(x)
+        outputs = tf.argmax(x, axis=-1)
+        model = keras.Model(inputs=inputs, outputs=outputs)
+        return model
+
+    def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
```
Comment (elad-c) [inline tests/keras_tests/feature_networks_tests/feature_networks/mixed_precision_tests.py:537]: Verify test fails  without the output replacemant

File: tests/pytorch_tests/model_tests/feature_models/mixed_precision_activation_test.py [hunk 2]
```diff
@@ -121,6 +121,21 @@ def compare(self, quantized_models, float_model, input_x=None, quantization_info
...
+        super().__init__(unit_test)
+        self.expected_config = [0, 0, 0, 0]
+
+    def get_kpi(self):
+        return KPI(np.inf, np.inf)
+
+    def create_feature_network(self, input_shape):
+        return MixedPrecisionOutputReplacementNet(input_shape)
+
+    def compare(self, quantized_models, float_model, input_x=None, quantization_info=None):
```
Comment (elad-c) [inline tests/pytorch_tests/model_tests/feature_models/mixed_precision_activation_test.py:135]: same as TF

**Review by elad-c — APPROVED** [review]

**Review by elad-c — APPROVED** [review]
